atm/VOLKOV/src/main/java/com/mybank/atm/controller/MyBank.java

```diff
@@ -0,0 +1,202 @@
+                            "Your balance " + currentClient.getCard().getAcount().getMoneyOnAcount() + "$\n" +
+                            "Good bye"
+
+            );
+            System.out.println(" ");
+            makeSelect();
+        } else if (selected == 3) {
+            takeMoney();
+
+        } else if (selected == 4) {
+            transferMoney();
+
+        } else if (selected == 5) {
+            System.out.println("Good Bye");
+            System.out.println(" ");
+            startUpATM();
+
+        } else {
+            makeSelect();
+        }
+
+
+    }
+
+    private void transferMoney() {
+        Scanner scanner = new Scanner(System.in);
+        int idNumToTransfer=0;
+        int moneyToTransfer=0;
+        System.out.println("Please enter card number of client to whom you want make transfer:");
+        if (scanner.hasNextInt()) {
+            idNumToTransfer = scanner.nextInt();
+        } else {
+            System.out.println("Wrong select \nTry again");
+            transferMoney();
+        }
+        Client clientToTransfer = clientsRepository.getClientByCardNum(idNumToTransfer);
+        if (clientToTransfer==null || clientToTransfer.getCard().getCardNum()==currentClient.getCard().getCardNum()){
+            System.out.println("Wrong card number");
+            transferMoney();
+        }
+        System.out.println("How much money you want to  transfer?");
+        if (scanner.hasNextInt()) {
+            moneyToTransfer = scanner.nextInt();
+        } else {
+            System.out.println("Wrong select \nTry again");
+            transferMoney();
+        }
+        if (idNumToTransfer!=0 && (moneyToTransfer>0 && moneyToTransfer<1+currentClient.getCard().getAcount().getMoneyOnAcount())) {
+            int moneyOnCurrentClientAcount = currentClient.getCard().getAcount().getMoneyOnAcount();
+            currentClient.getCard().getAcount().setMoneyOnAcount(moneyOnCurrentClientAcount-moneyToTransfer);
+            int moneyOnTransferClientAcount = clientToTransfer.getCard().getAcount().getMoneyOnAcount();
+            clientToTransfer.getCard().getAcount().setMoneyOnAcount(moneyOnTransferClientAcount+moneyToTransfer);
+            System.out.println("Your balance now is " +currentClient.getCard().getAcount().getMoneyOnAcount()+"$" );
+            System.out.println(clientToTransfer.getClientName() +" "+clientToTransfer.getClientSurName() + " balance now is " +clientToTransfer.getCard().getAcount().getMoneyOnAcount()+"$" );
+            System.out.println(" ");
+        }
+        else if (moneyToTransfer>currentClient.getCard().getAcount().getMoneyOnAcount()-1) {
+            System.out.println("Not enough money\nPleas try again");
+            transferMoney();
+        } else {
+            System.out.println("Something go wrong\nTry again");
+            transferMoney();
+        }
+        makeSelect();
```

Comment:
this method is not a part of the transfer money logic, should be placed into some other place